src/integrationTest/java/uk/gov/hmcts/darts/utils/IntegrationBase.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.darts.utils;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.springframework.boot.autoconfigure.ImportAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.webservices.server.WebServiceServerTest;
+import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
+import org.springframework.cloud.openfeign.FeignAutoConfiguration;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.test.context.ActiveProfiles;
+
+@WebServiceServerTest
+@ImportAutoConfiguration({FeignAutoConfiguration.class})
+@ComponentScan("uk.gov.hmcts.darts")
+@ActiveProfiles("int-test")
+@AutoConfigureWireMock(port = 8090)
+public class IntegrationBase {
+
+    protected EventApiStub theEventApi = new EventApiStub();
+    protected GetCourtLogsApiStub courtLogsApi = new GetCourtLogsApiStub();
+
+    @BeforeEach
+    void clearDbInThisOrder() {
+        theEventApi.clearStubs();
+        courtLogsApi.clearStubs();
```

Comment:
Do we need to call `WireMock.reset()` multiple times? As it's `static` I think the call has global effect, so may only need invoked once.